Desktop/Regression_test_for_the_Desktop_Client.md

```diff
@@ -0,0 +1,288 @@
+# Regression test for the Desktop Client
+
+## Previous requirements
+
+Have a Desktop Client vX.X.X ready to be used with owncloud.
+
+Prepare a OC server with ssl actived and trusted certificates.
+
+Have a LDAP server ready to be used with owncloud.
+
+Have an Active Directory server ready to be used with owncloud.
+
+Have ready two external storage of your choice (e.g S3)
+
+## Testing
+
+### 1. Settings windows - Login
+
+TestID | Test Case | Steps to reprouce |Expected Result |Result| Related Comment
+------------ | ------------- | ------------- | ------- | ----- | ------
+1 | Update Installation | 1. You need to have installed a previous version 2. Update the new version ||:construction:| |
+2 | Install the new version (with url: HTTPS) | 1. Delete the previous version 2. Install the new version || :construction: | |
+3 | Install the new version (with url: HTTP) | 1. Delete the previous version 2. Install the new version || :construction: | |
```

Comment:
What you want to check is a fresh install? okii, I'd only check with https, it shouldn't be any difference... trying to keep the regression handleable, i'd remove the 3